94-appendixD.Rmd

```diff
@@ -23,18 +23,18 @@ lc <- 0
 ```{r message=FALSE}
 library(dplyr)
 library(ggplot2)
-library(nycflights13)
+library(nycflights23)
 ```
 
-**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Repeat the above installing steps, but for the `dplyr`, `nycflights13`, and `knitr` packages. This will install the earlier mentioned `dplyr` package, the `nycflights13` package containing data on all domestic flights leaving a NYC airport in 2013, and the `knitr` package for writing reports in R. 
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Repeat the above installing steps, but for the `dplyr`, `nycflights23`, and `knitr` packages. This will install the earlier mentioned `dplyr` package, the `nycflights23` package containing data on all domestic flights leaving a NYC airport in 2023, and the `knitr` package for writing reports in R. 
```

Comment:
```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Repeat the above installing steps, but for the `dplyr`, `nycflights13`, and `knitr` packages. This will install the earlier mentioned `dplyr` package, the `nycflights13` package containing data on all domestic flights leaving a NYC airport in 2013, and the `knitr` package for writing reports in R. 
```